server/db/storage.ts

```diff
@@ -0,0 +1,105 @@
+import { createReadStream } from 'fs'
+import config from '../config'
+
+const azuretestcreds = [
+  'devstoreaccount1',
+  'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==',
+  'http://127.0.0.1:10000/devstoreaccount1',
+]
+
+interface IStorageProps {
+  backing?: 'azure' | 'aws'
+  endpoint?: string
+  region?: string
+  local?: boolean
+}
+
+class Storage {
```

Comment:
yeah wow we should extract this out to have two classes that both implement IStorageProps :|